packages/rocketchat-lib/server/functions/deleteMessage.js

```diff
@@ -15,12 +16,17 @@ RocketChat.deleteMessage = function(message, user) {
 		}
 	} else {
 		if (!showDeletedStatus) {
+			deletedMsg = RocketChat.models.Messages.findOneById(message._id);
```

Comment:
Needed to capture this message before it gets deleted so that I can fire it to interested parties